evap/student/templates/student_vote.html

```diff
@@ -312,6 +315,117 @@ <h3 class="mb-3">{{ evaluation.full_name }} ({{ evaluation.course.semester.name
             function textResultsPublishConfirmationBottomChanged() {
                 $("#text_results_publish_confirmation_top").prop('checked', $("#text_results_publish_confirmation_bottom").prop('checked'));
             }
+
+            function selectByNumberKey(row, num) {
+                if (num === 0) {
+                    // Select "No answer"
+                    num = row.children.length;
+                }
+                const index = num - 1;
+
+                if (!(0 <= index && index < row.children.length)) return;
+
+                const maybeNextElement = row
+                    .children[index]
+                    .getElementsByClassName("num-selectable");
+                if (maybeNextElement.length === 1) {
+                    maybeNextElement[0].click();
+                }
+            }
+
+            const studentForm = document.getElementById("student-vote-form");
+            const focussable = Array.from(studentForm.getElementsByClassName("tab-selectable"));
+            studentForm.addEventListener("keydown", e => {
+                const current = document.activeElement;
+                if (!["INPUT", "LABEL", "SPAN", "TEXTAREA", "BUTTON"].includes(current.tagName)) {
+                    // Don't do anything special
+                    return;
+                }
+
+                // Since the event could be caught on either the outer label or
+                // the nested text / input, the full row could be two steps up
+                let currentRow = current.parentElement;
+                if (currentRow.tagName !== "DIV") currentRow = currentRow.parentElement;
+                if (currentRow.tagName !== "DIV") {
+                    return;
+                }
+
+                const insideSubmitRow = currentRow.parentElement.classList.contains("card-submit-area");
```

Comment:
```suggestion
                const insideSubmitRow = currentRow.closest(".card-submit-area") !== null;
```
Just to be a bit more independent of the implemented markup structure.